run-tests.sh

```diff
@@ -16,7 +16,7 @@ if [[ "${1:-}" = "--in-vm" ]]; then
   export GOCACHE=/run/go-cache
 
   echo Running tests...
-  /usr/local/bin/go test -coverprofile="$1/coverage.txt" -covermode=atomic -v ./...
+  go test -coverprofile="$1/coverage.txt" -covermode=atomic -v ./...
```

Comment:
@lmb Not sure if this was an absolute path for a reason, but my Go executable is in another place.

Reply:
Great, looks like `go` is not in Semaphore's path, I'll revert this.